modules/database/variables.tf

```diff
@@ -5,13 +5,18 @@ variable "allowed_security_groups" {
 
 variable "database_credentials" {
   description = "Credentials to set for database master user"
-
   type = object({
     username = string,
     password = string,
   })
 }
 
+variable "kms_key_policy" {
+  description = "Custom Key Policy for the Aurora KMS Key"
+  type        = string
+  default     = ""
```

Comment:
would `null` be more reliable as a default?

Reply:
Yeah I think so, I now remember that we were moving away from empty quotes and using `null` a while back, updated